src/failed-spec-parser.js

```diff
@@ -2,14 +2,13 @@ export default function (output = '') {
   let match = null
   let CUCUMBERJS_TEST = /^\d+ scenarios?/m
   let failedSpecs = new Set()
-  let PROTRACTOR_SHARDED = /------------------------------------/g
+  let MULTI_TESTS = /------------------------------------/g
   let SPECFILE_REG = /.+Specs:\s(.*\.js)/g
 
-  if (PROTRACTOR_SHARDED.test(output) && SPECFILE_REG.test(output)) {
-    console.log('sharded test found')
-    let testsOutput = output.split('------------------------------------')
-    testsOutput.shift()
-    let RESULT_REG = /\d+\sspec|assertions?,\s(\d+)\sfailures?/g
+  if (MULTI_TESTS.test(output) && SPECFILE_REG.test(output)) {
+    let testsOutput = output.split('------------------------------------').slice(1)
```

Comment:
After taking a look at this it seems like we can avoid `slice`ing if we just add `if (specFile && result === 'failed')` on line 23. I may have missed the intention though